lms/templates/certificates/shipping_information.html

```diff
@@ -0,0 +1,81 @@
+<%page expression_filter="h"/>
+
+<%inherit file="../main.html" />
+
+<%block name="pagetitle">Certificate Shipping Information</%block>
+<script type="text/javascript">
+function editMode(){
+    var $editable = $('.editable');
+    var $readOnly = $('.read-only');
+    $readOnly.addClass('hidden');
+    $editable.removeClass('hidden');
+    console.log('called');
+};
+</script>
+
+<style>
+.read-only {
+    line-height: 25px;
+}
+
+#shipping_form {
+    margin: 25px;
+}
+
+label {
+    display: block;
+    float: left;
+    width: 100px;
+    font-style: normal;
+    font-weight: bold;
+    cursor: default;
+}
+
+.clearfix {
+    margin-bottom: 10px;
+}
+</style>
+
+    <h1>
+        Course Certificate Shipping Information
+    </h1>
+
+    <div id="shipping_form">
+        <form method="POST">
+            <input type="hidden" id="csrf_token" name="csrfmiddlewaretoken" value="${csrf_token}">
```

Comment:
Consider using Django's built-in Form class for generating and handling this! Might not be worth it for the experiment, but it could simplify things quite a bit!